pytorch_lightning/strategies/hpu.py

```diff
@@ -0,0 +1,78 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
please keep the filenames consistent: this should be `single_hpu` to align with `single_tpu` that already exists here: https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/strategies/single_tpu.py